doc/new_user.rst

```diff
@@ -4,7 +4,11 @@ New user basics
 Set up teuthology
 -----------------
 
-#. Log in to teuthology.front.sepia.ceph.com
+#. Request access via the process described at:
+   https://github.com/ceph/sepia/blob/master/doc/adding_users.rst
```

Comment:
I was just following the pattern of the rest of the doc; I haven't touched it before I don't think. I don't see ``:ref:`` used anywhere in this repo.